src/main/java/com/coveros/selenified/utilities/ReportOverview.java

```diff
@@ -0,0 +1,316 @@
+/*
+ * Copyright 2019 Coveros, Inc.
+ *
+ * This file is part of Selenified.
+ *
+ * Selenified is licensed under the Apache License, Version
+ * 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.coveros.selenified.utilities;
+
+import com.coveros.selenified.Browser;
+import org.testng.ISuite;
+import org.testng.ITestResult;
+import org.testng.internal.Utils;
+import org.testng.log4testng.Logger;
+import org.testng.reporters.EmailableReporter2;
+import org.testng.xml.XmlSuite;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.text.NumberFormat;
+import java.util.List;
+
+import static com.coveros.selenified.Selenified.REPORTER;
+import static com.coveros.selenified.utilities.Constants.*;
+import static com.coveros.selenified.utilities.Property.BROWSER;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.nio.file.Files.newBufferedWriter;
+
+/**
+ * Creates a custom report at the conclusion of the test run, which nicely lays
+ * out the high level results, and links to all of the detailed reports
+ *
+ * @author Max Saperstone
+ * @version 3.2.0
+ * @lastupdate 6/28/2019
+ */
+public class ReportOverview extends EmailableReporter2 {
+
+    private static final Logger log = Logger.getLogger(ReportOverview.class);
+    private static final String WARNING = "warning";
+    private static final String SUCCESS = "success";
+    private static final String DANGER = "danger";
+    private static final String TR = "</tr>";
+    private static final String fileName = "report.html";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS00115)